packages/verifier/src/analytics/constants.ts

```diff
@@ -47,5 +47,5 @@ export enum CustomEventNames {
 }
 
 // REMOVE name and number at end of Pilot
-export const PROPERTY_PATH_WHITELIST = ['name', 'number', 'valid', 'lang', 'countryCode']
+export const PROPERTY_PATH_WHITELIST = ['name', 'number', 'valid', 'lang', 'countryCode', 'issuer']
```

Comment:
I think you meant to add this to mobile app, not to the verifier:
https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/analytics/constants.ts#L166

Reply:
🙉